reference/versions/0.5.0/reference.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
So, I can't add new analysis without updating references or without testing it. How can I proceed to implement new analysis?.